the_mines/get_html.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import shutil
+import os
+
+
+def get_html(matchday, season):
+
+    # - The URL of this website uses only the latter year in a season (2019/2020).
+    # - season[5:9] selects the latter season.
+    # - I'm using the 2019/2020 convention for season input because I believe it
+    #   is what most users are familiar with when asked for a season
```

Comment:
Thanks for the info! You could wrap this up into a multiline comment, but I think we can save that for issue #15.